### PR TITLE
Prep javascript operators and expressions

### PR DESCRIPTION
**Pull request by devtanole**
Screenshot 2024-06-20 at 2 12 57 AM

### CODE REVIEW

File: prep-javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+console.log('type of fullName:', typeof fullName);
+const pH = 8;
+let isAcidic = pH < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('type of isAcidic:', typeof isAcidic);
+const numberOfSoldiers = 301;
+let isSparta = numberOfSoldiers === 300;
+console.log('value of isSparta:', isSparta);
+console.log('type of isSparta:', typeof isSparta);
+let nickname = fullName + ' is the GOAT';
```
Comment (robgmerrill) [inline prep-javascript-operators-and-expressions/main.js]: Close! The exercise asks that you use the "assignment operator"

```
let nickname = fullName;
nickname += ' is the GOAT';
```

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Good start! Small change requested.

File: prep-javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (devtanole) [reply prep-javascript-operators-and-expressions/main.js]: I have updated the file.

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**